Makefile.include

```diff
@@ -78,22 +78,23 @@ LDFLAGS  += -L$(TOP_DIR) \
 ifeq ($(EMULATOR),1)
 CFLAGS   += -DEMULATOR=1
 
-ifeq ($(HEADLESS),1)
-CFLAGS   += -DHEADLESS=1
-else
-CFLAGS   += -DHEADLESS=0
-
-CFLAGS   += $(shell pkg-config --cflags sdl2)
-LDLIBS   += $(shell pkg-config --libs sdl2)
-endif
-
 CFLAGS   += -include $(TOP_DIR)emulator/emulator.h
 CFLAGS   += -include stdio.h
 
 LDFLAGS  += -L$(TOP_DIR)emulator
 
 LDLIBS   += -ltrezor -lemulator
 LIBDEPS  += $(TOP_DIR)/libtrezor.a $(TOP_DIR)emulator/libemulator.a
+
+ifeq ($(HEADLESS),1)
+CFLAGS   += -DHEADLESS=1
+else
+CFLAGS   += -DHEADLESS=0
+
+CFLAGS   += $(shell sdl2-config --cflags)
```

Comment:
NACK at the moment, because it breaks the build on my setup 😆 

TREZOR Core uses `-LSDL2 -LSDL2_Image` and it seems to work for everyone, so maybe we should switch to that for now

https://github.com/trezor/trezor-core/blob/30396f9cf03be20a6198ad1f76139e97c9fcc664/SConscript.unix#L97

Reply:
Oh I did not want to leave this in, left it by mistake. (That's why git is showing it differently.)

Removing it for now